components/raftstore/src/store/fsm/peer.rs

```diff
@@ -140,7 +143,7 @@ where
     raft_entry_max_size: f64,
     batch_req_size: u32,
     request: Option<RaftCmdRequest>,
-    callbacks: Vec<(Callback<E::Snapshot>, usize, TiInstant)>,
```

Comment:
Why remove `TiInstant`?

Reply:
Because there is a bug in waterfall metrics here. If the cmd is not batched, the request time in its callback is not set.
So we don't need the `TiInstant` to set the request time in callback(it will be set during construction).